citar.el

```diff
@@ -99,21 +98,46 @@ to include."
   :type '(repeat string))
 
 (defcustom citar-templates
-  '((main . "${author editor:30}     ${date year issued:4}     ${title:48}")
-    (suffix . "          ${=key= id:15}    ${=type=:12}    ${tags keywords keywords:*}")
-    (preview . "${author editor} (${year issued date}) ${title}, ${journal journaltitle publisher container-title collection-title}.\n")
-    (note . "Notes on ${author editor}, ${title}"))
+  '((main . "%->30a     %4d     %->48t")
```

Comment:
```suggestion
  '((main . "%->30a     %.4d     %->48t")
```

Reply:
Actually, this isn't right either: if there is no date, then it doesn't leave 4 blank spaces, therefore, alignment problems.

I think it should be `%>4d`

Reply:
> I think it should be `%>4d`

Correct.